strapi/strapi
# Make add URL work with pending step

### What does it do?

Add assets now fills the pending step of the add asset dialog

### Why is it needed?


### How to test it?

- Go to the ML
- Click Add new asset
- Select "From url"
- Add some url (one per line)
- Validate
- You should see a preview of the different files

### Screen

![2021-09-16 15 31 55](https://user-images.githubusercontent.com/3874873/133621384-7aa38a58-52fb-48ec-92f5-9ee153234641.gif)
